src/core/Akka.Cluster.Tests.MultiNode/ClusterAccrualFailureDetectorSpec.cs

```diff
@@ -30,6 +30,8 @@ public ClusterAccrualFailureDetectorMultiSpec()
             CommonConfig= DebugConfig(false)
                             .WithFallback(ConfigurationFactory.ParseString("akka.cluster.failure-detector.threshold = 4"))
                             .WithFallback(MultiNodeClusterSpec.ClusterConfig());
+
+            TestTransport = true;
```

Comment:
Just needed to turn the `TestTransport` on in order to get the specs to pass; otherwise Throttling / Blackholing doesn't work